core/tabs/system-setup/arch/linux-neptune.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/sh -e
```

Comment:
```suggestion
#!/bin/sh -e
```
keep the consistency with other scripts